### ISSUE TITLE
Default Favicon

### ISSUE THREAD

**Issue by DhairyaBahl** [hash-define-organization/Hash-Whiteboard#7]
### Description
Currently the app is having default favicon which is provided in create react app. Replace it with the icon of Hash Define. 

### Suggestion
You can also be creative and can also propose your own favicon. 

### Note
This issue is totally dependent on the creativity and not the skill of the person so that is why I won't assign this to anyone ! Just propose your ideas for favicons with actual icon and the best one will get selected.

**Comment by Prajnaprabhu3** [comment]
Can I work on this issue

**Comment by DhairyaBahl** [comment]
> Can I work on this issue

Yes Sure ! Be creative and create your own favicon ;) If its good then we will definitely use it.

**Comment by Aniruddha-Inge** [comment]
@DhairyaBahl I have raised a pull request regarding this issue here -> https://github.com/hash-define-organization/Hash-Whiteboard/pull/15 
Please have a look :)

**Comment by anmolchhabra21** [comment]
Hello, @DhairyaBahl I can contribute on this issue also under GSSOC'22 so please assign me this if it is not resolved yet.
